docs/src/introduction_to_catalyst/introduction_to_catalyst.md

```diff
@@ -1,301 +1,168 @@
+* The simulation's timespan. That is, the time frame over which we wish to run the simulation.
+* The simulation's parameter values. That is, the values of the model's parameters for this simulation.
+
+The initial conditions and parameter values are declared as vectors of pairs. Here, each entry pairs the symbol corresponding to a specific quantity's name to its value (e.g. `:S => 99` denotes that the initial value of `S` should be `99`). The timespan is simply a tuple with the simulations *starting* and *final* times. Once we have set these values, we collect them in a so-called `ODEProblem`.
+```@example intro_1
+u0 = [:S => 99, :I => 1, :R => 0]
+tspan = (0.0, 20.0)
+ps = [:β => 0.025, :γ => 0.2]
+oprob = ODEProblem(sir_model, u0, tspan, ps)
 ```
-Alternatively, we can use ModelingToolkit-based symbolic species variables to
-specify these mappings like
-```@example tut1
-t = default_t()
-@parameters  α K n δ γ β μ
-@species m₁(t) m₂(t) m₃(t) P₁(t) P₂(t) P₃(t)
-psymmap  = (α => .5, K => 40, n => 2, δ => log(2)/120,
-         γ => 5e-3, β => 20*log(2)/120, μ => log(2)/60)
-u₀symmap = [m₁ => 0., m₂ => 0., m₃ => 0., P₁ => 20., P₂ => 0., P₃ => 0.]
-nothing   # hide
+To simulate our `ODEProblem`, we simply input it to the `solve` command. Before we can do so, however, we also need to activate the [OrdinaryDiffEq.jl](https://github.com/SciML/OrdinaryDiffEq.jl) package (which is used for all ODE simulations in Julia).
+```@example intro_1
+using OrdinaryDiffEq
+sol = solve(oprob)
+```
+Finally, we can plot the solution using the [Plots.jl](https://github.com/JuliaPlots/Plots.jl) package's `plot` function:
+```@example intro_1
+using Plots
+plot(sol)
 ```
-Knowing these mappings we can set up the `ODEProblem` we want to solve:
-
-```@example tut1
-# time interval to solve on
-tspan = (0., 10000.)
 
-# create the ODEProblem we want to solve
-oprob = ODEProblem(repressilator, u₀map, tspan, pmap)
-nothing   # hide
+### [SDE simulations](@id introduction_to_catalyst_model_simulation_sde)
+SDE simulations can be carried out in a very similar manner to ODE ones, but by creating an `SDEProblem` instead of an `ODEProblem`. However, we can re-use the problem inputs from previously:
+```@example intro_1
+sprob = SDEProblem(sir_model, u0, tspan, ps)
+nothing # hide
 ```
-By passing `repressilator` directly to the `ODEProblem`, Catalyst has to
-(internally) call `convert(ODESystem, repressilator)` again to generate the
-symbolic ODEs. We could instead pass `odesys` directly like
-```@example tut1
-odesys = complete(odesys)
-oprob2 = ODEProblem(odesys, u₀symmap, tspan, psymmap)
-nothing   # hide
+Next, we import the [StochasticDiffEq.jl](https://github.com/SciML/StochasticDiffEq.jl) package (which is used for all SDE simulations in Julia) and use it to simulate our SDE. 
+```@example intro_1
+using StochasticDiffEq
+sol = solve(sprob, STrapezoid())
+sol = solve(sprob, STrapezoid(); seed = 1234) # hide
```

Comment:
Generally, I pu hidden seed simulations throughout the docs so that they look the same each time they are built. However, I agree that we should not depend on this. For all cases I have done this, I go through and ensure that the stochastic behaviour is more or less the same (e.g. I would not repeat what we had previously and have a workflow that depends on a seed in the early stage, but which might give a different result if this is modified).